scripts/generatePackage.js

```diff
@@ -69,5 +65,6 @@ const packagesFolder = './packages';
         process.exit(1);
     }
 
+    // eslint-disable-next-line no-console
```

Comment:
Should not be necessary

Reply:
True https://github.com/trezor/trezor-suite/pull/5751/commits/4bb811986b6e11032801424f316c647fba27656e